### ISSUE TITLE
Create rule S6807 (`jsx-a11y/role-has-required-aria-props`): DOM elements with ARIA roles should have the required properties

### ISSUE THREAD

**Issue by ilia-kebets-sonarsource** [SonarSource/SonarJS#4243]
[plugin rule](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/role-has-required-aria-props.md#jsx-a11yrole-has-required-aria-props)
